benchmarks/sparse/dlmc/utils.py

```diff
@@ -0,0 +1,141 @@
+import torch
+from pathlib import Path
+import sys
+from scipy import sparse
+import math 
+from scipy.sparse import isspmatrix
+
+def to_coo_scipy(x):
+    indices_1 = x._indices().numpy()
+    values_1 = x._values().numpy()
+    return sparse.coo_matrix((values_1, (indices_1[0], indices_1[1])),
+                             shape=x.shape)
+
+def scipy_coo_matmul(mat1, mat2):
+    if isspmatrix(mat1) and isspmatrix(mat2):
+        result = mat1.dot(mat2).tocoo()
+        return torch.sparse_coo_tensor([result.row, result.col], result.data,
```

Comment:
why are you converting result to torch coo tensor? Would that distort scipy benchmarking results?

Reply:
Not needed actually, I simplified this to `mat1.dot(mat2)`